app/bt/AffectedUserFlow/verificationAPI.ts

```diff
@@ -0,0 +1,118 @@
+import Config from 'react-native-config';
+
+const baseUrl = Config.GAEN_VERIFY_URL;
+const verifyUrl = `${baseUrl}/api/verify`;
+const certificateUrl = `${baseUrl}/api/certificate`;
+
+const defaultHeaders = {
+  'content-type': 'application/json',
+  accept: 'application/json',
+  'X-API-Key': Config.GAEN_VERIFY_API_TOKEN,
```

Comment:
these header keys _could_ be constants, might be handy if we're constructing headers elsewhere

Reply:
totally, I would expect that this would be the only bit that needs it though as we will only have the two endpoints to hit for the verification flow.